Sources/SwiftProtobuf/Message+JSONAdditions.swift

```diff
@@ -22,9 +22,13 @@ public extension Message {
   /// serializing to JSON.
   ///
   /// - Returns: A string containing the JSON serialization of the message.
+  /// - Parameters:
+  ///   - options: The JSONEncodingOptions to use.
   /// - Throws: `JSONEncodingError` if encoding fails.
-  func jsonString() throws -> String {
-    let data = try jsonUTF8Data()
+  func jsonString(
+    options: JSONEncodingOptions = JSONEncodingOptions()
```

Comment:
How do we want to handle this from a library version number pov?

Reply:
This change (and the change to `jsonUTF8Data()` below) are certainly backwards source compatible.

But I'm not sure if adding a new optional parameter retains backwards binary compatibility.  @allevato would know.

If it's source and binary backwards compatible, then I see no implications at all for version numbering.

If it's not binary compatible, then it deserves at least a minor version update.

Reply:
Added a commit to bump the version to 1.2.0 now so we don't forget.